Unix/doc/setup-ntlm-omi.md

```diff
@@ -177,9 +177,92 @@ On the command line:
 
 ##Enabling SPNEGO/NTLM on the OMI server
 
+Since it is the most secure, by default, only NTLM authentication via winbind is configured in omiserver. There may be reasons
+why this would not be desirable for a particular implementation, in which case it is possible to define a credential file,
+using the same format as described for the client. 
 
-The most secure method for providing omiserver credentials is via winbind. There may be reasons why this would not be 
-practical for a particular implementation, in which case it is possible to define a credential file, using the same format as
-described for the client.  If this method is used, the location of the file is specified in the omiserver.conf file.
+If this method is used, use of the file is specified by setting the option 
+```
+NtlmCredsFile= *path* 
```

Comment:
Since this is effectively documentation, please make the above line the **suggested** entry, and not what the server would normally accept.

I think, in general, we recommend `tag=value` without a space.

BTW, since this isn't quoted, would spaces be accepted in the filename? You should test that and, if not, say something here about that (or fix it so they are, but that can be tricky without accepting quote characters in case of leading/trailing spaces).